### DISCUSSION TITLE
Global Modeling

### DISCUSSION THREAD

**Discussion by diegulio** [ourownstory/neural_prophet#531] [Q&A - get help using NeuralProphet] [answered]
Hello, This is such a nice framework! Has global modeling been implemented? I haven't found any documentation about it.  Thanks in advance!

**Comment by ourownstory** [comment, marked as answer]
Hi @diegulio Thank you for your anticipation of the Global modelling feature. 
A basic version is already live, but @MateusGheorghe is making it more useful with #507 and he will also provide an example notebook in #499. 
It should be merged by the end of the week.

**Reply by diegulio** [reply]
Thank you very much for your answer! I'll be looking forward to the merge! :D
